Failed install on Ubuntu 18.04 and also Debian 11

I tried installing PolyCube from source on Ubuntu 18-04 and Debian 11 but it keeps failing while throwing this error:
<<ListKey.cpp >>
-------------------------
$HOME_DIR/polycube/src/polycubed/src/server/Resources/Body/ListKey.cpp: In static member function ‘static polycube::polycubed::Rest::Resources::Body::ListType polycube::polycubed::Rest::Resources::Body::ListKey::FromYang(LY_DATA_TYPE)’:
$HOME_DIRpolycube/src/polycubed/src/server/Resources/Body/ListKey.cpp:83:16: error: ‘runtime_error’ is not a member of ‘std’
   83 |     throw std::runtime_error("Unrecognized YANG data type");
      |                ^~~~~~~~~~~~~
$HOME_DIR/polycube/src/polycubed/src/server/Resources/Body/ListKey.cpp:85:1: warning: control reaches end of non-void function [-Wreturn-type]
   85 | }
      | ^
make[2]: *** [src/polycubed/src/CMakeFiles/polycubed.dir/build.make:576: src/polycubed/src/CMakeFiles/polycubed.dir/server/Resources/Body/ListKey.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:5529: src/polycubed/src/CMakeFiles/polycubed.dir/all] Error 2

Any idea  ?
Thank you in advance

[ 76%] Building CXX object src/polycubed/src/CMakeFiles/polycubed.dir/server/Validators/UnionValidator.cpp.o
/home/ubuntu/polycube/src/polycubed/src/server/Resources/Body/ListKey.cpp: In static member function ‘static polycube::polycubed::Rest::Resources::Body::ListType polycube::polycubed::Rest::Resources::Body::ListKey::FromYang(LY_DATA_TYPE)’:
/home/ubuntu/polycube/src/polycubed/src/server/Resources/Body/ListKey.cpp:83:16: error: ‘runtime_error’ is not a member of ‘std’
   83 |     throw std::runtime_error("Unrecognized YANG data type");
      |                ^~~~~~~~~~~~~
/home/ubuntu/polycube/src/polycubed/src/server/Resources/Body/ListKey.cpp:85:1: warning: control reaches end of non-void function [-Wreturn-type]
   85 | }
      | ^
make[2]: *** [src/polycubed/src/CMakeFiles/polycubed.dir/build.make:557: src/polycubed/src/CMakeFiles/polycubed.dir/server/Resources/Body/ListKey.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/home/ubuntu/polycube/src/polycubed/src/server/Resources/Data/Lib/Service.cpp: In member function ‘virtual Response polycube::polycubed::Rest::Resources::Data::Lib::Service::WriteValue(const string&, const json&, const ListKeyValues&, polycube::polycubed::Rest::Resources::Endpoint::Operation)’:
/home/ubuntu/polycube/src/polycubed/src/server/Resources/Data/Lib/Service.cpp:82:1: warning: control reaches end of non-void function [-Wreturn-type]
   82 | }
      | ^
make[2]: Leaving directory '/home/ubuntu/polycube/build'
make[1]: *** [CMakeFiles/Makefile2:2962: src/polycubed/src/CMakeFiles/polycubed.dir/all] Error 2
make[1]: Leaving directory '/home/ubuntu/polycube/build'
make: *** [Makefile:130: all] Error 2
++ error_message
++ set +x

Error during installation
***********************SYSTEM INFO*************************************
kernel version: 5.8.0-59-generic
linux release info: DISTRIB_ID=Ubuntu DISTRIB_RELEASE=20.10 DISTRIB_CODENAME=groovy DISTRIB_DESCRIPTION="Ubuntu 20.10" NAME="Ubuntu" VERSION="20.10 (Groovy Gorilla)" ID=ubuntu ID_LIKE=debian PRETTY_NAME="Ubuntu 20.10" VERSION_ID="20.10" HOME_URL="https://www.ubuntu.com/" SUPPORT_URL="https://help.ubuntu.com/" BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/" PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy" VERSION_CODENAME=groovy UBUNTU_CODENAME=groovy
g++ version: g++ (Ubuntu 10.3.0-1ubuntu1~20.10) 10.3.0 Copyright (C) 2020 Free Software Foundation, Inc. This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
gcc version: gcc (Ubuntu 10.3.0-1ubuntu1~20.10) 10.3.0 Copyright (C) 2020 Free Software Foundation, Inc. This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
golang-version: go version go1.14.7 linux/amd64

ubuntu@xxx:~/polycube$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.10
Release:        20.10
Codename:       groovy
ubuntu@xxx:~/polycube$ uname -a
Linux p1 5.8.0-59-generic #66-Ubuntu SMP Thu Jun 17 00:46:01 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

--- a/src/polycubed/src/server/Resources/Body/ListKey.cpp
+++ b/src/polycubed/src/server/Resources/Body/ListKey.cpp
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 The Polycube Authors
+ * Copyright 2018-2021 The Polycube Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 #include "ListKey.h"
 
 #include <memory>
@@ -20,7 +21,7 @@
 #include <typeindex>
 #include <utility>
 #include <vector>
-#include <stdexcept>        // g++-10 requires to explicitly include stdexcept when using exceptions
+#include <stdexcept>        // g++ v10 requires to explicitly include stdexcept when using exceptions
 
 namespace polycube::polycubed::Rest::Resources::Body {
 ListKey::ListKey(

--- a/src/polycubed/src/server/Resources/Body/ListKey.cpp
+++ b/src/polycubed/src/server/Resources/Body/ListKey.cpp
@@ -20,6 +20,7 @@
 #include <typeindex>
 #include <utility>
 #include <vector>
+#include <stdexcept>
 
 namespace polycube::polycubed::Rest::Resources::Body {
 ListKey::ListKey(

--- a/src/polycubed/src/server/Resources/Body/ListKey.cpp
+++ b/src/polycubed/src/server/Resources/Body/ListKey.cpp
@@ -20,7 +20,7 @@
 #include <typeindex>
 #include <utility>
 #include <vector>
-#include <stdexcept>
+#include <stdexcept>        // g++-10 requires to explicitly include stdexcept when using exceptions
 
 namespace polycube::polycubed::Rest::Resources::Body {
 ListKey::ListKey(

